src/app/code/community/Shopgate/Cloudapi/Model/Api2/Customers/Rest/Customer/V2.php

```diff
@@ -31,6 +31,15 @@ class Shopgate_Cloudapi_Model_Api2_Customers_Rest_Customer_V2 extends Shopgate_C
     protected function _retrieve()
     {
         $customerData = Mage::getModel('customer/customer_api')->info($this->getApiUser()->getUserId());
+        $customerData['addresses'] = array();
+
+        if ($customerData['default_billing']) {
```

Comment:
eee, don't you need to do an `isset` or `!empty` in this language? 🍶

Reply:
In case no address is set, it is always null, e. g. 
"default_shipping": null,
I could do a null check, but I wanted to save characters. That's why I was sticking with a simple if clause. :)